specs/block_spec.rb

```diff
@@ -0,0 +1,87 @@
+require_relative './spec_helper.rb'
+
+describe "Wave 3: Blocks" do
+  describe "Initialize" do
+
+    it "can initialize a block" do
+      new_hotel = Hotels::Hotel.new
+      dates = new_hotel.check_dates([2020, 9, 10], [2020, 9, 12])
+      date_range = new_hotel.date_range(dates)
+      number_of_rooms = 5
+      blocks = new_hotel.make_blocks(date_range, number_of_rooms)
+      new_block = Hotels::Block.new(date_range, blocks)
+
+
+      new_block.must_be_instance_of Hotels::Block
+
+    end
+  end
+  describe "make_reservation" do
+
+    it "can reserve a room from a block" do
```

Comment:
There's a lot going on here that's not related to a Block.  Instead it's highly involved with hotel.  Block should be more independent of Hotel and test separately.